eng/pipelines/templates/stages/vmr-validation.yml

```diff
@@ -83,6 +87,10 @@ stages:
       parameters:
         targetArchitecture: arm64
         OS: Linux
+        reuseBuildArtifactsFrom:
+        # Scenario-tests package is built/published by Windows_x64 job
+        - Windows_x64
```

Comment:
These are a lot of assets (IIRC > 4GB the last time I looked). Could we instead live build the scenario-tests package or its assemblies live in the jobs that need it?

Reply:
That was the alternative option, but there are issue with it. As @mthalman was also investigating this for another feature, it is not simple to just build the scenario-tests repo, without various dependent repos. This would just increase the overall VMR build time.

Current PR is an improvement compared to we have in place now. We were downloading all merged Blob and Package artifacts, which amount to ~30GB. With this PR, we will download ~6GB.

Reply:
Also, there are many packages that are not published from Linux jobs, as they are rid-agnostic. To get a complete set of packages we need to download artifacts from `Windows_x64` job.

Reply:
> Also, there are many rid-agnostic packages that are not published from Linux jobs. To get a complete set of packages we need to download artifacts from Windows_x64 job.

Makes sense. Ok if you need the complete set of rid-agnostic nuget packages anyway that might get downloaded during scenario-tests execution, then this looks good.

What about the packages that get produced in the other verticals? Don't you need those, i.e. the linux RID specific ones that aren't available in the pre-restored SDK "packs" folder?

Reply:
All checks are green - if you're ok with this, can you approve the PR?